mkosi/distributions/kali.py

```diff
@@ -24,11 +24,15 @@ def default_tools_tree_distribution(cls) -> Distribution:
         return Distribution.kali
 
     @classmethod
-    def repositories(cls, context: Context, local: bool = True) -> Iterable[AptRepository]:
-        if context.config.snapshot:
+    def repositories(cls, context: Context, for_image: bool = False) -> Iterable[AptRepository]:
+        mirror = None if for_image else context.config.mirror
+        if not mirror:
+            mirror = "http://http.kali.org/kali"
```

Comment:
That should be mentioned in the man page in the table for default mirrors (it should have been there before, but I just noticed).

Reply:
Huh? I can see it right there in the table already

Reply:
I guess I was blind yesterday.